rescript/plugin_setup.py

```diff
@@ -641,19 +641,25 @@
     parameters={
         'query': Str,
         'accession_ids': Metadata,
-        'entrez_delay': Float,
-        'levels': List[Str]},
+        'ranks': List[Str],
+        'allowed_ranks': List[Str],
+        'disable_rank_propagation': Bool,
```

Comment:
Wouldn't it be easier to change this to `enable_rank_propagation` and `no_enable_propagation`? 
If feels odd to say the following as a double-negative : `no_disable_rank_propagation`. Also potentially harder for a user to understand / interpret which of those two choices is `False`.

Reply:
good call!

Reply:
I phrased it like that because `no_disable_rank_propagation` is the default and `no_enable_propagation` don't read good. So a user would only ever enter `--p-disable-rank-propagation`, which seems more self-explanatory.

But I can change it if you want.

Reply:
how about `rank_propagation` and `no_rank_propagation`?

Reply:
Elegant and to-the-point. _We likes it precious. We likes it!_

Reply:
That's the winner. Thanks Nick.